.github/workflows/test.yml

```diff
@@ -8,14 +8,14 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: [ '2.5', '2.6', '2.7', '3.0', '3.1', '3.2', jruby ]
+        ruby: [ '3.0', '3.1', '3.2', '3.3', head, jruby ]
```

Comment:
Sorry for responding late 🙇‍♂️ 
Would you mind keeping 2.x Rubies? If this was an app it's fine, but libraries, including carrierwave-aws, are better be permissive on its dependencies I think, as users may have a reason that prevents from upgrading.

Reply:
Thank you for your reply. I'm not sure what the consensus is on unmaintained Ruby versions. I'm obviously not in a position to decide that. But I think that supporting old versions also leads to people not updating their software. In the end that means that they are running insecure software. Also, people with old Ruby versions might still be using old versions of this gem. Maybe someone else has an opinion on this topic. I can also just add the old versions back, if that's the prevailing opinion.